modules/kafka/src/main/scala/com.snowplowanalytics.snowplow.enrich.kafka/Main.scala

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2022 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+
+package com.snowplowanalytics.snowplow.enrich.kafka
+
+import cats._
+import cats.implicits._
+import cats.effect.{ExitCode, IO, IOApp, Resource, SyncIO}
+import com.snowplowanalytics.snowplow.enrich.common.fs2.{Run, Telemetry}
+import fs2.kafka.CommittableConsumerRecord
+
+import scala.concurrent.ExecutionContext
+
+import java.util.concurrent.{Executors, TimeUnit}
+import com.snowplowanalytics.snowplow.enrich.kafka.generated.BuildInfo
```

Comment:
Yeah imports aren't well organized but scalafmt is happy, modifying scalafmt would lead to a giant PR here so I'll manually fix it for now. I'd prefer to fix imports in another PR

Reply:
You don't need to modify scalafmt to reorganize the imports, scalafmt doesn't care much about them (with our configuration)

Reply:
fixed them manually for today